# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/acl-uclibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- acl-2.2.39/setfacl/Makefile.orig	2006-06-20 08:51:25.000000000 +0200
+++ acl-2.2.39/setfacl/Makefile	2007-01-17 15:34:27.000000000 +0100
@@ -10,6 +10,9 @@
 HFILES = sequence.h parse.h
 
 LLDLIBS = $(LIBACL) $(LIBATTR) $(LIBMISC)
+ifeq ($(ENABLE_GETTEXT),yes)
+LLDLIBS += /usr/lib/libintl.a
+endif
 LTDEPENDENCIES = $(LIBACL) $(LIBMISC)
 
 default: $(LTCOMMAND)
